packages/nc-gui/components/general/Overlay.vue

```diff
@@ -11,6 +11,7 @@ interface Props {
   target?: TeleportProps['to']
   teleportDisabled?: TeleportProps['disabled']
   transition?: boolean
+  lightBackground?: boolean
```

Comment:
This is unnecessary, you can just add class names to the component already.

If you need a lighter background you can use `!bg-opacity-75` as `class` on the component.

Reply:
Resolved my convos. This approach is more clean. However, I think it would be `!bg-opacity-25` instead assuming the default background is `bg-gray-700/75`.

Reply:
Depends on the color
if you use a white background and lower the opacity to 25, it will appear like a dark background because white with very low opacity is not super light but rather dark 😄